mk/source.mk

```diff
@@ -308,7 +307,6 @@ COMMON_SRC += \
             drivers/accgyro/accgyro_spi_mpu6500.c \
             drivers/accgyro/accgyro_spi_mpu9250.c \
             drivers/accgyro/accgyro_virtual.c \
-            drivers/accgyro/gyro_sync.c \
```

Comment:
This file appears in both COMMON_SRC and ifneq ($(SIMULATOR_BUILD),yes) section:
drivers/accgyro/gyro_sync.c

Found another duplicate:

- Line appears twice in SPEED_OPTIMISED_SRC:
rx/frsky_crc.c

- Partial duplicates - these files appear in both COMMON_SRC and RX_SRC:
drivers/rx/rx_pwm.c
drivers/rx/rx_spi.c
drivers/rx/rx_xn297.c